cypress/e2e/ephemeral/ee.cy.ts

```diff
@@ -0,0 +1,10 @@
+describe('Ephemeral test', () => {
+  beforeEach(() => {
+    // cy.login();
```

Comment:
Is there any reason not to remove the entire `beforeEach`?

Reply:
Sorry this was not supposed to be here, I was doing some POC with Matt about Konflux PR checks. I'll remove this file.